### PR TITLE
Minor changes to fix test failures.

### LINKED ISSUE #1070: Fix HapiClientValidator for the failing Random/Hotspot Performance tests
**Summary of the defect**
HapiClientValidator fails for various errors while running Random Performance tests. Fix the validator to accommodate the errors.

Failing test link : https://hedera-hashgraph.slack.com/archives/C018Y4E6ADT/p1612774191035600
https://hedera-hashgraph.slack.com/archives/C018Y4E6ADT/p1612778939035800

### PR DESCRIPTION
**Pull request by ljianghedera**
**Related issue(s)**:
Closes #1070

slack report: https://hedera-hashgraph.slack.com/archives/C018CBQBTGB/p1614308585078800 
https://hedera-hashgraph.slack.com/archives/C018CBQBTGB/p1614293992071600

Note: there changes should fix some failure scenarios. More changes may be needed. 

**Summary of the change**:
1. Changes to fix these test failures.

**External impacts**:
None.

**Applicable documentation**
- [ ] Javadoc
- [ ] HAPI protobuf comments
- [ ] README

### CODE REVIEW

File: test-clients/src/main/java/com/hedera/services/bdd/suites/perf/SubmitMessageLoadTest.java [hunk 1]
```diff
@@ -195,7 +197,8 @@ private static HapiApiSpec runSubmitMessages() {
 						TOPIC_EXPIRED,
 						INVALID_TOPIC_ID,
 						INSUFFICIENT_PAYER_BALANCE)
-				.hasKnownStatusFrom(SUCCESS, OK, INVALID_TOPIC_ID)
+				.hasKnownStatusFrom(SUCCESS, OK, INVALID_TOPIC_ID, INSUFFICIENT_PAYER_BALANCE
+						,UNKNOWN,TRANSACTION_EXPIRED)
```
Comment (Neeharika-Sompalli) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/perf/SubmitMessageLoadTest.java:201]: Why is UNKNOWN added as Known status?

File: test-clients/src/main/java/com/hedera/services/bdd/suites/perf/TokenTransfersLoadProvider.java [hunk 2]
```diff
@@ -102,7 +109,9 @@ private HapiApiSpec runTokenTransfers() {
 						// end to prevent it from making new export files that may cause account balances validator to
 						// be inconsistent. The freeze shouldn't cause normal perf test any issue.
 						freeze().payingWith(GENESIS)
-								.startingIn(10).seconds()
+								.startingIn(30).seconds()
+								.hasKnownStatusFrom(SUCCESS,UNKNOWN)
```
Comment (Neeharika-Sompalli) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/perf/TokenTransfersLoadProvider.java:113]: same here

File: test-clients/src/main/java/com/hedera/services/bdd/suites/perf/SubmitMessageLoadTest.java (hunk 1 above)
Comment (ljianghedera) [reply test-clients/src/main/java/com/hedera/services/bdd/suites/perf/SubmitMessageLoadTest.java:201]: This part is tricky. I am not sure if some recent changes is causing this or not. In several test runs, the client logs has marked the `UNKNOWN` as reason for suite failure by the test suite. And the summary of the suite FAILED is picked up by client validator (we can also let the HAPIClientValidator to ignore this message, which I am not sure is a good idea or not).  IMO, at end of the transaction duration, there should be a clear decision for this transaction. The `UNKNOWN` status shouldn't appear as the response to the test client at all. I think until we have a larger scope discussion on the error handling, we won't have a clear answer for this. 

Or if anyone has a better suggestion, I will be glad to follow.

File: test-clients/src/main/java/com/hedera/services/bdd/suites/perf/TokenTransfersLoadProvider.java [hunk 3]
```diff
@@ -221,8 +231,9 @@ private HapiApiSpec runTokenTransfers() {
 						}
 					}
 					op = cryptoTransfer(xfers)
-							.hasKnownStatusFrom(NOISY_ALLOWED_STATUSES)
+							.hasKnownStatusFrom(OK, DUPLICATE_TRANSACTION, SUCCESS, UNKNOWN, INSUFFICIENT_PAYER_BALANCE)
 							.hasRetryPrecheckFrom(NOISY_RETRY_PRECHECKS)
+							.hasPrecheckFrom(OK, PLATFORM_NOT_ACTIVE)
```
Comment (anighanta) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/perf/TokenTransfersLoadProvider.java:236]: Why is PLATFORM_NOT_ACTIVE acceptable ?

Comment (ljianghedera) [reply test-clients/src/main/java/com/hedera/services/bdd/suites/perf/TokenTransfersLoadProvider.java:236]: If the platform is still not in `ACTIVE` state, the client will get the this value as response code.

Comment (ljianghedera) [reply test-clients/src/main/java/com/hedera/services/bdd/suites/perf/TokenTransfersLoadProvider.java:236]: This is the way EET framework communicates to test application what happened to this request. It's part of error processing already. For the JRS test design, when the test client sends the request to server, normally, the server should be already in `ACTIVE` state. However, for whatever reason, occasionally the server may get delayed and may get into the `ACTIVE` state later and process future requests. We'll still consider the overall test legitimate.

**Review by qnswirlds — COMMENTED** [review]
Each load/perf test usually starts with creating a sender/payer account with recharging option. I think adding `INSUFFICIENT_PAYER_BALANCE` into `hasKnownStatusFrom` will turn off this option, hence when the account runs out of balance, it will not be recharged and we will receive more `INSUFFICIENT_PAYER_BALANCE` until the test finishes. The `INSUFFICIENT_PAYER_BALANCE` return code could also be received when the `test payer` runs out balance to recharge the mentioned sender/payer account.

**Comment by ljianghedera** [comment]
> Each load/perf test usually starts with creating a sender/payer account with recharging option. I think adding `INSUFFICIENT_PAYER_BALANCE` into `hasKnownStatusFrom` will turn off this option, hence when the account runs out of balance, it will not be recharged and we will receive more `INSUFFICIENT_PAYER_BALANCE` until the test finishes. The `INSUFFICIENT_PAYER_BALANCE` return code could also be received when the `test payer` runs out balance to recharge the mentioned sender/payer account.

No, I think this will turn off the recharging of the (default) payer. I could be wrong, my understanding is recharging for the  payer happens based on the recharging window and it should always happen with this pace. While other transaction will happen in the meantime. If other transactions happen in high volume, there exists the chance that the pay doesn't have enough balance to pay for some transactions before it's recharged. The `INSUFFICIENT_PAYER_BALANCE` status should cover this scenario.

File: test-clients/src/main/java/com/hedera/services/bdd/suites/perf/TokenTransfersLoadProvider.java (hunk 3 above)
Comment (JeffreyDallas) [reply test-clients/src/main/java/com/hedera/services/bdd/suites/perf/TokenTransfersLoadProvider.java:236]: A long approach would be a change to JRS, only launches client after the swirlds.log shows the platform is in ACTIVE mode, so we can avoid the timing issue between launching server and launching client

Comment (ljianghedera) [reply test-clients/src/main/java/com/hedera/services/bdd/suites/perf/TokenTransfersLoadProvider.java:236]: That will be a better solution. But this check is still necessary in cases where network glitches may happen.

**Review by anighanta — APPROVED** [review]

**Pull request merged**